### PR TITLE
Remove goog.functions.and from ol.interaction.Modify

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: src/ol/interaction/modifyinteraction.js [hunk 1]
```diff
@@ -113,14 +112,21 @@ ol.interaction.Modify = function(options) {
   });
 
   /**
+   * @private
+   * @param {ol.MapBrowserEvent} mapBrowserEvent Browser event.
+   * @return {boolean} Combined condition result.
```
Comment (marcjansen) [inline src/ol/interaction/modifyinteraction.js]: Return type `ol.events.ConditionType`?

**Comment by marcjansen** [comment]
One minor question, otherwise this looks good to me. Please merge at will with or without the question adressed (assuming checks turn green)

File: src/ol/interaction/modifyinteraction.js (hunk 1 above)
Comment (fredj) [inline src/ol/interaction/modifyinteraction.js]: It's `{boolean}`; the function returns the results from the functions invocation.

Comment (marcjansen) [inline src/ol/interaction/modifyinteraction.js]: Yeah, I was missreading the (now thankfully gone `@type`). Thanks

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**